content/articles/pathlib-vs-os-modules-for-file-and-directory-system-operation/index.md

```diff
@@ -0,0 +1,919 @@
+- It tends to be utilized to execute a file irrespective of the extension (e.g csv, pdf, mp3, and so on)
+  but unfortunately, it only works on Windows operating except you use some other available module python recommends.
+- It provides different functions that can perform additional operations on environment variables
+  like setting an environment, removing an environment, and more operations.
+- It provides a capacity that can copy a document.
+
+#### Pathlib
+- It gives high-level Path object for performing operations on pathname and class path has methods that enable the
+  capacity of performing I/O activity which is much simpler and
+  behaves like the in-built open function thus python documentation recommends.
+- It offers distinguished classes for performing explicit system operations on various operating systems
+  which clearly expresses that the pathlib module isn't portable.
+- Its module objects' methods generally return system compatible objects (e.g PurePathPosix, PosixPath, PurePathWindows,
+  and WindowsPath) which enables us to perform the additional operations.
+- It doesn't give either article or strategy to making transitory records and catalogs.
+- For listing a directory content when using pathlib is a tedious task because pathlib only provides method iterator
+  of the class path for listing directory content which requires additional operation because method iterator returns an
+  object representation.
+  - Class path in pathlib provides a method for creating a permanent empty file with different extension which makes
+    working with File System Operation mind-boggling in python.
+- It can't be utilized to execute a document yet can open a record while composing or perusing a .txt document.
+- It doesn't make any arrangement for environment variables operation.
+- None of the available objects in pathlib makes a provision for making a duplicate file.
+
+### Practical difference between OS and Pathlib module
+In this section i will be illustrating the semantic differences between Pathlib and Os module, with comparable code
+snippets to help you determine where every module fit in. I realized that there are many conducive differences which may
+not be absolutely understood without seeing how it works, additionally, note that every demonstration requires the
+importation of the two modules.
+
+### Querying current path using os and pathlib
+Querying the current working directory is a basic task when you are dealing with a path.  There are two unique types of
+path which are relative path and absolute path.An absolute path gives a full description of the current location you are
+working while a relative path refers to a location that is relative to the current working directory. Note querying the
+current path using os and pathlib will return an absolute path.
+
+Querying current path using os.
+```python
+import os
+
+os.getcwd()
```

Comment:
Are you trying this out with notebook? Use print statements

Reply:
Yes sir
Okay
I will use print instead, but there's a reason I use notebook for some specific code snippets.